### PR TITLE
NPC dialogue fixes

### PR DESCRIPTION
**Pull request by DataCrusade**
Fixed a few places where the old version of a renamed item was mentioned (Item Magnet -> Dowsing Machine, Shard Case -> Gem Case), and minor grammar/punctuation tweaks in a few places

### CODE REVIEW

File: src/scripts/towns/TownList.ts [hunk 1]
```diff
@@ -158,11 +158,11 @@ const PalletProfOak = new ProfNPC('Prof. Oak',
 
 const PalletMom1 = new NPC('Mom', [
     'So you\'re really leaving on your very own Pokémon journey. I\'m so proud of you. Let me give you some words of wisdom for your travels.',
-    'Traveling on your own can be scary. But remember that there are nice people everywhere you go. So strike up a converstation. You will probably learn something useful.',
+    'Traveling on your own can be scary. But remember that there are nice people everywhere you go. So strike up a converstation! You will probably learn something useful.',
```
Comment (PixLSteam) [inline src/scripts/towns/TownList.ts:161]:
```suggestion
    'Traveling on your own can be scary. But remember that there are nice people everywhere you go. So strike up a conversation! You will probably learn something useful.',
```
Might as well fix this typo while we're at it :)

File: src/scripts/towns/TownList.ts [hunk 2]
```diff
@@ -158,11 +158,11 @@ const PalletProfOak = new ProfNPC('Prof. Oak',
...
 ],
 { requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('Tutorial Quests', 2), new QuestLineStepCompletedRequirement('Tutorial Quests', 3, GameConstants.AchievementOption.less )]) });
 const PalletMom2 = new NPC('Mom', [
-    'Remember that there are nice people everywhere you go. So strike up a converstation. You will probably learn something useful.',
+    'Remember that there are nice people everywhere you go. So strike up a converstation! You will probably learn something useful.',
```
Comment (PixLSteam) [inline src/scripts/towns/TownList.ts:165]:
```suggestion
    'Remember that there are nice people everywhere you go. So strike up a conversation! You will probably learn something useful.',
```
See above.

File: src/scripts/towns/TownList.ts [hunk 3]
```diff
@@ -703,7 +703,7 @@ const CianwoodPhotographyAide = new NPC('Photography Aide', [
 ]);
 
 const MahoganySouvenirShopAttendant = new NPC('Souvenir Shop Attendant', [
-    'We’ve got stuff here nobody else has got! But keep any Item Magnets you have away from the merchandise… especially the RageCandyBars. Keep ‘em outside where they belong! I’ve heard magnets can attract Pokémon with held items more often, and even more so in Dungeons!',
+    'We’ve got stuff here nobody else has got! But keep any Dowsing Machines you have away from the merchandise… especially the Rage Candy Bars. Keep ‘em outside where they belong! I’ve heard those machines can attract Pokémon with held items more often, and even more so in Dungeons!',
```
Comment (PixLSteam) [inline src/scripts/towns/TownList.ts:706]:
```suggestion
    'We’ve got stuff here nobody else has got! But keep any Dowsing Machines you have away from the merchandise… especially the RageCandyBars. Keep ‘em outside where they belong! I’ve heard those machines can attract Pokémon with held items more often, and even more so in Dungeons!',
```
This one is a reference: https://bulbapedia.bulbagarden.net/wiki/Rage_Candy_Bar

**Review by PixLSteam — CHANGES_REQUESTED** [review]
Looks good overall, just some minor feedback 👍

**Comment by DataCrusade** [comment]
> Looks good overall, just some minor feedback 👍

Good catch @PixLSteam, completely missed that typo (was already there, I just didn't see it). Wasn't sure on the RageCandyBar thing so I thought I'd change it and if it was intentional, someone would let me know. Thanks.

**Review by Jaaslet — APPROVED** [review]
Looks good. Thanks for the PR

**Pull request merged**